content/en/docs/ops/diagnostic-tools/component-logging/index.md

```diff
@@ -20,9 +20,9 @@ Logging messages output by a component are categorized by *scopes*. A scope repr
 you can control as a whole. Different components have different scopes, depending on the features the component
 provides. All components have the `default` scope, which is used for non-categorized log messages.
 
-As an example, as of this writing, `istioctl` has 24 scopes, representing different functional areas within the command:
+As an example, as of this writing, `istioctl` has 25 scopes, representing different functional areas within the command:
```

Comment:
Is there a reason we want to show the scopes here? It means we have to routinely verify what the scopes are. I'm not sure if we can just point to the reference for these as well.

Also, `cli` was removed from this list, but is used in an example command line further down the page. We should update that command line as well.

Reply:
> Is there a reason we want to show the scopes here? It means we have to routinely verify what the scopes are. I'm not sure if we can just point to the reference for these as well.

Not sure, but I think this is just for the quick reference of scopes. otherwise, the user will have to run the controlz dashboard to see the logging scopes.

> Also, cli was removed from this list, but is used in an example command line further down the page. We should update that command line as well.

Thanks, I will update it.

Reply:
Done.

Reply:
What I was expecting was that you might have changed the example command to remove the missing scopes.  Right now if I run the command it fails with a missing scope so it shouldn't be added:
```
> istioctl version
no running Istio pods in "istio-system"
1.9.0-beta.0
> istioctl analyze --log_output_level attributes:debug,cli:warn 
Error: unknown scope 'attributes' specified
```

Reply:
From the command line:
```
istioctl options help
The following options can be passed to any command:
... where scope can be one of [ads, adsc, all, analysis, authn, authorization, ca, cache, cli, default, installer, klog, mcp, model, patch, processing, resource, source, spiffe, tpath, translator, util, validation, validationController, wle] ...
```

Reply:
Done.